src/wilson/dom.cljs

```diff
@@ -174,9 +174,15 @@
   (update elem 1 merge-attrs attrs))
 
 (defn with-class
-  "Adds a class to an element."
-  [cls elem]
-  (with-attrs {:class cls} elem))
+  "Adds a class to a component."
+  [cls component]
+  (if (vector? component)
```

Comment:
I think we're thinking of different call signatures, I'm trying to make sure that `(with-class "yolo" [thingy-component a b c])` works; so "vector" doesn't tell you much?